### DISCUSSION TITLE
Query: some dependencies (eg ESP_DoubleResetDetector) have been archived- safe to keep using?

### DISCUSSION THREAD

**Discussion by julianrendell** [khoih-prog/ESPAsync_WiFiManager#132] [Q&A]
Hi - thanks for your libraries!

They came up when Google searching for using a Captive Portal WiFi Manager for ESP32 + double reset detection.

Another thankyou for the many examples available to refer to - including how to use the double reset detection library.

But I see that the DRD library has been archived- is it because it's "stable" and bugs aren't expected? i.e. would you recommend still using it for new projects?

Thanks in advance!

**Comment by khoih-prog** [comment]
HI @julianrendell 

**Archived** library only means that library is **read-only** now. The reason is either

- it's quite **stable**
- not intended to be updated
- **not popular** and not intended to spend more time

For me, because of there are too many (200+) of my libraries, I couldn't continue working / maintain and create new libraries without **distraction against too many rude and/or lazy questions / issues**.

As you know and experience in embedded **programming** world, we can be **better everyday** by doing a lot of **active** works and experiments, and learn from fixing errors by **research** ourselves. Only asking for help after those experiments, failures to fix, etc., by **not surrender immediately**.

Therefore I archived many of them to avoid **unnecessary** interactions. If there is some reason to update, I'll unarchive and update.

But don't worry, those libraries will continue to be maintained / updated as needed. For example the **DRD**, etc.

The question / issue can be posted on **Arduino Forum**, etc. so that many other experts there can help and I'm aware of it to fix bug or update as **needed.**

You can continue to use them as normal.

Thanks for your nice words and usage of the libraries,

BR,
